src/ngraph/pass/opset1_downgrade.cpp

```diff
@@ -88,6 +91,38 @@ bool pass::Opset1Downgrade::run_on_node(shared_ptr<Node> node)
         modified = true;
         break;
     }
+    case OP_TYPEID::Reverse:
+    {
+        auto tmp = as_type_ptr<op::v1::Reverse>(node);
+        auto axes_node = tmp->input_value(1).get_node_shared_ptr();
+        NGRAPH_CHECK(axes_node->is_constant(),
+                     "Unable to convert Reverse:v1 to Reverse:v0 "
+                     "if reduction axes are not constant. Node: ",
+                     *node);
+        const auto axes_node_const = dynamic_pointer_cast<op::Constant>(axes_node);
```

Comment:
If we know that axes_node is constant could we use static_pointer_cast here?
```suggestion
        const auto axes_node_const = static_pointer_cast<op::Constant>(axes_node);
```

Reply:
as_type_ptr?

Reply:
I changed it to `as_type_ptr`